hw_diag/app.py

```diff
@@ -40,7 +40,7 @@ def get_app(name):
     @scheduler.task(
         'cron',
         id='run_diagnostics',
-        minute='5,10,15,20,25,30,35,40,45,50,55')
+        minute='*')
```

Comment:
No, this wont' work. The reason why we have these explicit minutes (every 5, except `:00`) is because we have a custom job at the top of the hour. Hence if we just add `*` this will change the behavior. 

I will chat with @robputt how to change this and avoid the scheduler all together for everything but the actual GCS job as we really don't need the scheduler now that we've reworked things a fair bit.

Reply:
@vpetersson the scheduler can run multiple jobs at the same time.

Reply:
Sure but there is really no need for a scheduler anymore. We are wasting a lot of CPU cycles on this. Might as well remove it as per https://github.com/NebraLtd/hm-diag/issues/174. This will both speed it up too, as we don't need to wait for potentially 59 seconds for it to run.